src/adagrams.js

```diff
@@ -1,8 +1,135 @@
 const Adagrams = {
+  
   drawLetters() {
-    // Implement this method for wave 1
+    const rawTiles = {
```

Comment:
Style wise - I recommend putting the object outside of the `drawLetters()` function (but inside the module).